web/html/src/manager/content-management/shared/business/filters.enum.js

```diff
@@ -1,128 +1,143 @@
-      matcher: 'contains',
-      text: 'Package (contains Name)',
-    },
-    PACKAGE_NEVRA: {
-      key: 'package_nevra',
-      entityType: 'package',
-      matcher: 'equals',
-      text: 'Package (matches NEVRA)',
-    },
-    ERRATUM: {
-      key: 'erratum',
-      entityType: 'erratum',
-      matcher: 'equals',
-      text: 'Patch (matches Advisory Name)',
-    },
-    ERRATUM_BYTYPE: {
-      key: 'erratum_bytype',
-      entityType: 'erratum',
-      matcher: 'equals',
-      text: 'Patch (matches Advisory Type)',
-    },
-    ERRATUM_BYDATE: {
-      key: 'erratum_bydate',
-      entityType: 'erratum',
-      matcher: 'greatereq',
-      text: 'Patch (issued after)',
-    },
-    ERRATUM_BYSYNOPSIS: {
-      key: 'erratum_bysynopsis',
-      entityType: 'erratum',
-      matcher: 'equals',
-      text: 'Patch (equals Synopsis)',
-    },
-    ERRATUM_BYSYNOPSIS_CONTAINS: {
-      key: 'erratum_bysynopsis_contains',
-      entityType: 'erratum',
-      matcher: 'contains',
-      text: 'Patch (contains Synopsis)',
-    },
-    ERRATUM_PKG_NAME: {
-      key: 'erratum_pkg_name',
-      entityType: 'erratum',
-      matcher: 'contains_pkg_name',
-      text: 'Patch contains package name',
-    },
-    ERRATUM_PKG_LT_EVR: {
-      key: 'erratum_pkg_lt_evr',
-      entityType: 'erratum',
-      matcher: 'contains_pkg_lt_evr',
-      text: 'Patch contains package with version lower than',
-    },
-    ERRATUM_PKG_LE_EVR: {
-      key: 'erratum_pkg_le_evr',
-      entityType: 'erratum',
-      matcher: 'contains_pkg_le_evr',
-      text: 'Patch contains package with version lower or equal than',
-    },
-    ERRATUM_PKG_EQ_EVR: {
-      key: 'erratum_pkg_eq_evr',
-      entityType: 'erratum',
-      matcher: 'contains_pkg_eq_evr',
-      text: 'Patch contains package with version equal',
-    },
-    ERRATUM_PKG_GE_EVR: {
-      key: 'erratum_pkg_ge_evr',
-      entityType: 'erratum',
-      matcher: 'contains_pkg_ge_evr',
-      text: 'Patch contains package with version greater or equal than',
-    },
-    ERRATUM_PKG_GT_EVR: {
-      key: 'erratum_pkg_gt_evr',
-      entityType: 'erratum',
-      matcher: 'contains_pkg_gt_evr',
-      text: 'Patch contains package with version greater than',
-    }
-  },
-  objectDefaultValueHandler(defaultState)
-);
+export type ClmFilterOptionType = {key: string, entityType: FilterEntityType, matchers: Array<FilterMatcherType>, text: string}
+type ClmFilterOptionsEnumType = { [key: string]: ClmFilterOptionType }
 
-function getFiltersOptions(): Array<FilterOptionType> {
-  return (Object.values(filtersOptionsEnum): any);
-}
+export const filterEntity : FilterEntityEnumType = {
+  PACKAGE: {
+    key: 'package',
+    text: t('Package')
+  },
+  ERRATUM: {
+    key: 'erratum',
+    text: t('Patch')
+  }
+};
 
-function findByKey(key: string): FilterOptionType {
-  return _find(filtersOptionsEnum, entry => entry.key === key) || defaultState;
-}
+export const filterMatchers : FilterMatcherEnumType = {
```

Comment:
I don't see any usage of this const out of this module - do we need to export it?